src/awsexplorer/regionNode.ts

```diff
@@ -37,10 +37,15 @@ export class RegionNode extends AWSTreeNodeBase {
     public get regionCode(): string {
         return this.region.id
     }
+
     public get regionName(): string {
         return this.region.name
     }
 
+    public determineRegion(): string {
+        return this.regionCode
```

Comment:
is this different than `regionCode()` (line 37)?

Reply:
It's only different in that I gave it a different name. I needed to have it be `determineRegion` so that it overwrites the default method for `determineRegion` in the AWSBaseTreeNode class. Is there another way to do this without duplicating code?

Reply:
Oh I figured there was an advantage to having nodes without `regionCode` as a property, but I guess I can't really think of a use case for that in the explorer. I'll go ahead and try that approach.